packages/thread-context/tests/connection-proxy-test.js

```
@@ -0,0 +1,65 @@
+  const { createConnectionProxy } = require('meteor/thread-context');
+  const proxy = createConnectionProxy('conn-123');
+
+  test.equal(proxy.id, 'conn-123');
+});
+
+Tinytest.add('thread-context - ConnectionProxy - DDP property throws BridgeContextError', function (test) {
+  const { createConnectionProxy } = require('meteor/thread-context');
+  const { BridgeContextError } = require('meteor/thread-context');
+  const proxy = createConnectionProxy('conn-123');
+
+  test.throws(() => {
```

ESLint

[error] Insert `····`

(prettier/prettier)